lib/mrkt/concerns/custom_objects.rb

```diff
@@ -0,0 +1,54 @@
+module Mrkt
+  module CustomObjects
+    def get_list_of_custom_objects(object_names)
+      params = {
+        names: object_names.join(',')
+      }
+
+      get('/rest/v1/customobjects.json', params)
+    end
+
+    def describe_custom_object(object_name)
+      fail Mrkt::Errors::Unknown if !object_name || object_name.empty?
+
+      get("/rest/v1/customobjects/#{object_name}/describe.json")
+    end
+
+    def createupdate_custom_objects(object_name, objects, action: 'createOrUpdate', dedupe_by: 'dedupeFields')
+      post("/rest/v1/customobjects/#{object_name}.json") do |req|
+        params = {
+          input: objects,
+          action: action,
+          dedupeBy: dedupe_by
+        }
+        json_payload(req, params)
+      end
+    end
+
+    def delete_custom_objects(object_name, search_values, delete_by: 'dedupeFields')
+      post("/rest/v1/customobjects/#{object_name}/delete.json") do |req|
+        params = {
+          input: search_values,
+          deleteBy: delete_by
+        }
+
+        json_payload(req, params)
+      end
+    end
+
+    def get_custom_objects(object_name, filter_values, filter_type: 'dedupeFields', fields: nil, next_page_token: nil, batch_size: nil)
```

Comment:
`filterType` is required, so we should also require that, that should be the second argument after `name`